src/shim.c

```diff
@@ -971,6 +1028,7 @@ print_usage(void) {
         printf("  -c,  --container-id   Container id\n");
         printf("  -d,  --debug          Enable debug output\n");
         printf("  -t,  --token          Connection token passed by cc-proxy\n");
+        printf("  -r,  --rtimeout       Reconnection timeout to cc-proxy in seconds\n");
```

Comment:
About the long option name, I think you can go with something longer like `reconnect-timeout`